dashboard/client/average_response_time/average_response_time.js

```diff
@@ -0,0 +1,75 @@
+import d3 from 'd3';
+import nvd3 from 'nvd3';
+import _ from 'lodash';
+
+Template.averageResponseTime.onRendered(function () {
+  // Get reference to template instance
+  const templateInstance = Template.instance();
+
+  // Placeholder block
+  templateInstance.chartData = new ReactiveVar();
+
+  // Parse chart data reactively
+  templateInstance.autorun(() => {
+    const elasticsearchData = Template.currentData().elasticsearchData;
+
+    if (elasticsearchData) {
+      // Get aggregations from Elasticsearch data
+      const aggregatedData = elasticsearchData.aggregations.requests_over_time.buckets;
```

Comment:
This aggregation is slightly different than the original `requests_over_time`, so should have a more descriptive name.